src/components/presentational/SingleDataPoint/SingleDataPoint.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { CSSProperties } from 'react';
+import './SingleDataPoint.css';
+import { UnstyledButton } from '..';
+import { ColorDefinition } from '../../../helpers/colors';
+
+export interface SingleDataPointProps {
+    label: string;
+    statusText?: string;
+    statusColor?: ColorDefinition;
+    value?: number;
+    units?: string;
+    onClick?: () => void;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: SingleDataPointProps) {
+    const getContents = () => {
+        return <>
+            <div className="mdhui-single-data-point-value-units">
+                {!props.value &&
+                    <div className="mdhui-single-data-point-value">...</div>
+                }
+                {props.value &&
```

Comment:
When the value is 0 (which could be common if this were used for something like steps) it does weird stuff

![Screenshot 2024-02-07 at 4 27 12 PM](https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/862fa1b1-a758-4fe0-9e40-5852627b3dd3)

(I would probably default to treating 0 as a "no data" state...that is what most of the device data queriers do and I think the chart does...because often for Fitbit for instance data we get 0s when someone is not wearing a device)

Reply:
However that logic is maybe best handled upstream from here actually.  Is there a reason we just don't have value as a string?  That might make the most sense for this presentational component since I think this same component could be useful for non-numeric things

Reply:
No reason.  Sounds reasonable to me.  I've change the type of the `value` prop to be a `string`.